.github/workflows/ci.yml

```diff
@@ -111,6 +111,54 @@ jobs:
         flags: unit
         fail_ci_if_error: false
 
+  test-partial-backends:
```

Comment:
I wonder if it'd be simpler to just add an extra step to the existing test job, which will probably help to keep all the coverage steps together etc.

Reply:
I don't think a matrix will add much. We can just uninstall both libraries and do 1 extra test run.

Reply:
Sure, will update.